config/dispatcher/dispatcher.ex

```diff
@@ -17,29 +17,6 @@ defmodule Dispatcher do
     |> send_resp( 200, "{ \"message\": \"ok\" }" )
   end
 
-  ###############
-  # STATIC
-  ###############
-  match "/assets/*path", %{ layer: :static } do
-    forward conn, path, "http://frontend/assets/"
-  end
-
-  match "/forms/*path", %{ layer: :static } do
-    forward conn, path, "http://frontend/forms/"
-  end
-
-  get "/@appuniversum/*path", %{ layer: :static } do
-    forward conn, path, "http://frontend/@appuniversum/"
-  end
-
-  match "/index.html", %{ layer: :static } do
-    forward conn, [], "http://frontend/index.html"
-  end
-
-  match "/favicon.ico", %{ layer: :static } do
-    send_resp( conn, 404, "" )
-  end
-
```

Comment:
Looked at 2 other applications and they are  not using these static routes.
* https://github.com/lblod/app-centrale-vindplaats/blob/master/config/dispatcher/dispatcher.ex has  assets, flavicon defined
* https://github.com/lblod/app-digitaal-loket/blob/master/config/dispatcher/dispatcher.ex has no static paths defined

As there where no errors when running the application without these paths defined I removed them.